test/mocks/mock-search-responses.ts

```diff
@@ -31,6 +31,7 @@ export const getMockSuccessSingleResult: () => Result<
       },
     },
     rawResponse: {},
+    sessionContext: {},
```

Comment:
Just updating our mock responses to include empty session context, for type correctness & completeness' sake.

Reply:
I appreciate that bc then things match